src/core/Internal.ml

```diff
@@ -1712,24 +1719,24 @@ let solve
         let nconf = if restarts then to_int !n_of_conflicts else max_int in
         search env ~gc ~time ~memory ~progress nconf
       with
-        | () -> ()
-        | exception Restart ->
-          (* garbage collect clauses, if needed *)
-          if gc &&
-             !n_of_learnts >= 0. &&
-             float(Vec.size env.clauses_learnt - Vec.size env.trail) >= !n_of_learnts
-          then (
-            let n = (to_int !n_of_learnts) + 1 in
-            gc_clauses env ~down_to:n;
-          );
-
-          (* increment parameters to ensure termination *)
-          n_of_conflicts := !n_of_conflicts *. env.restart_inc;
-          n_of_learnts   := !n_of_learnts *. env.learntsize_inc;
-          (* diminish by how much n_of_learnts increases *)
-          env.learntsize_inc <- 1. +. (env.learntsize_inc -. 1.) /. 1.3 ;
-          loop()
-        | exception Sat -> check_sat ()
+      | () -> ()
```

Comment:
is this really produced by ocp-indent‽ I thought the config required 2 spaces…

Reply:
Yes it is. 
I think it is logical: the `begin` produces a new block, but the match doesn't.
You cannot indent two times.

Reply:
Thanks for making me discover the interrobang 💯 💯 💯